common-aggregator/src/main/java/com/redhat/cloud/notifications/models/AggregationCronjobParameters.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.cloud.notifications.models;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Objects;
+
+import static com.fasterxml.jackson.annotation.JsonFormat.Shape.STRING;
+import static com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy;
+
+@Entity
+@Table(name = "aggregation_cronjob_parameter")
+@JsonNaming(SnakeCaseStrategy.class)
```

Comment:
You will find many JPA entities with Jackson annotations in the code because we chose a long time ago to use a single layer of classes for the DB and the REST API. However, I've never been a big fan of that decision because it makes everything more complicated when we have to implement something, especially when it involves multiple Maven modules. For this specific case, I'd prefer if we had the JPA entity separated from the POJO that will be used for the REST API. The entity would stay in `common-aggregator` because it is need in `backend` and `aggregator` while the POJO for the REST API would only be in `backend`.

Reply:
Ok